src/main/java/filter/expression/Qualifier.java

```diff
@@ -898,4 +969,53 @@ private void applyState(TurboIssue issue) throws QualifierApplicationException {
     public String getName() {
         return name;
     }
+
+    private Qualifier replaceMilestoneAliasQualifier(List<TurboMilestone> allMilestones, int currentIndex) {
+        String contents = content.get();
```

Comment:
We could move the check for presence here. That will ensure that callers of this method never forget to check.